packages/Upgrade/src/api/services/ScheduledJobService.ts

```diff
@@ -5,13 +5,21 @@ import { ScheduledJobRepository } from '../repositories/ScheduledJobRepository';
 import { ScheduledJob, SCHEDULE_TYPE } from '../models/ScheduledJob';
 import { Experiment } from '../models/Experiment';
 import { EXPERIMENT_STATE } from 'ees_types';
+import AWS from 'aws-sdk';
+import { env } from '../../env';
+
+const stepFunction = new AWS.StepFunctions(
+  {
+    region: 'us-east-1',
```

Comment:
I think we should take this from environment variable

Reply:
Updated